### ISSUE TITLE
qamel docker gen vendor failed: permission denied

### ISSUE THREAD

**Issue by fcying** [RadhiFadlillah/qamel#11]
```
hello-world ‹master›$ qamel docker linux -o build/hello
Run `qamel build` from Docker image.
Starting build process.

Load config file...done
Generating vendor files...
Failed to vendor app: exit status 1
go: github.com/RadhiFadlillah/qamel@v0.0.0-20190708111841-8f5bb1aab5a7: mkdir /home/user/go/pkg: permission denied
go: error loading module requirements

Failed to build app using Docker: exit status 1
```

If I run `qamel build` first, and then run `qamel docker linux`, it will be fine.
```
hello-world ‹master›$ qamel build                      
Starting build process.

Load config file...done
Generating vendor files...done
Removing old build files...done
Generating binding files...done
Generating Qt resource file...done
Generating code for QML objects...done
Building app...done

Build finished succesfully.


hello-world ‹master›$ qamel docker linux -o build/hello
Run `qamel build` from Docker image.
Starting build process.

Load config file...done
Removing old build files...done
Generating binding files...done
Generating Qt resource file...done
Generating code for QML objects...done
Building app...done

Build finished succesfully.
```

**Comment by RadhiFadlillah** [comment]
Hi @fcying, could you check it out again. It should be fixed by [3dc4c5a4](https://github.com/RadhiFadlillah/qamel/commit/3dc4c5a41da0a2745a8da1efecd024e29ee93f1b)

**Comment by fcying** [comment]
It can work now~
